tests/fsharpqa/Source/Conformance/PatternMatching/Named/E_ActivePatterns01.fs

```diff
@@ -1,17 +1,11 @@
 // #Regression #Conformance #PatternMatching #ActivePatterns 
 // Verify error if Active Patterns do not start with an upper case letter
-//<Expects id="FS0623" status="error" span="(11,7)">Active pattern case identifiers must begin with an uppercase letter</Expects>
-//<Expects id="FS0623" status="error" span="(11,16)">Active pattern case identifiers must begin with an uppercase letter</Expects>
-//<Expects id="FS0623" status="error" span="(12,7)">Active pattern case identifiers must begin with an uppercase letter</Expects>
-//<Expects id="FS0623" status="error" span="(13,10)">Active pattern case identifiers must begin with an uppercase letter</Expects>
-//<Expects id="FS0623" status="error" span="(14,7)">Active pattern case identifiers must begin with an uppercase letter</Expects>
-//<Expects id="FS0624" status="error" span="(15,7)">The '\|' character is not permitted in active pattern case identifiers</Expects>
-//<Expects id="FS0624" status="error" span="(16,9)">The '\|' character is not permitted in active pattern case identifiers</Expects>
+//<Expects id="FS0623" status="error" span="(8,7)">Active pattern case identifiers must begin with an uppercase letter</Expects>
+//<Expects id="FS0623" status="error" span="(8,16)">Active pattern case identifiers must begin with an uppercase letter</Expects>
+//<Expects id="FS0624" status="error" span="(9,7)">The '\|' character is not permitted in active pattern case identifiers</Expects>
+//<Expects id="FS0624" status="error" span="(10,9)">The '\|' character is not permitted in active pattern case identifiers</Expects>
 
 let (|positive|negative|) n = if n < 0 then positive else negative
-let (|`` A``|) (x:int) = x
-let (|B1|``+B2``|) (x:int) = if x = 0 then OneA else ``+B2``
-let (|`` C``|_|) (x:int) = if x = 0 then Some(x) else None
```

Comment:
Would it be better to preserve these cases? So, if produced errors change one day it'll be reflected here.

Reply:
@auduchinok once we do the is letter thingy they will be errors again :-).

Reply:
Yes, it'll be OK. :)

What I wanted to say was if there is a test for some particular behavior that is being changed it could be good to keep the test just in case it might be changed again (even accidentally, e.g. if fixing an issue produced by the initial change). It's probably just a good thing to record how the particular behavior is changed anyway.

Reply:
Thanks @auduchinok mate, I understood mate, was messin' with ya :-)